<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookItem</class>
 <widget class="QWidget" name="BookItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="book_title_label">
        <property name="text">
         <string>book_title_label</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="book_info_label">
        <property name="text">
         <string>book_info_label</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="book_price_label">
        <property name="text">
         <string>book_price_label</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="book_amount_label">
        <property name="text">
         <string>book_amount_label</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="book_context_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>book_context_label</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="book_image_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>book_image_label</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
